javascript-forms/form-values.js

```diff
@@ -0,0 +1,12 @@
+var $form = document.getElementById('contact-form');
+
+var $submittedForm = {};
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  $submittedForm.name = $form.elements.name.value;
+  $submittedForm.email = $form.elements.email.value;
+  $submittedForm.message = $form.elements.message.value;
+
```

Comment:
The only thing missing here is that you are not resetting the form after submission. Make sure to view the link in the instructions on how to properly reset the form